src/main/java/uk/gov/register/service/RegisterService.java

```diff
@@ -15,18 +16,20 @@
     private final DBI dbi;
     private final MemoizationStore memoizationStore;
     private final ItemValidator itemValidator;
+    private final RegisterFieldsConfiguration registerFieldsConfiguration;
 
     @Inject
-    public RegisterService(RegisterData registerData, DBI dbi, MemoizationStore memoizationStore, ItemValidator itemValidator) {
+    public RegisterService(RegisterData registerData, DBI dbi, MemoizationStore memoizationStore, ItemValidator itemValidator, RegisterFieldsConfiguration registerFieldsConfiguration) {
         this.registerData = registerData;
         this.dbi = dbi;
         this.memoizationStore = memoizationStore;
         this.itemValidator = itemValidator;
+        this.registerFieldsConfiguration = registerFieldsConfiguration;
```

Comment:
Good question - I'm trying to avoid the number of registrations in `RegisterApplication` that don't need to be there. However you need it to be there for injection to `SearchResource` so I'm happy to leave it.